sound/soc/sof/loader.c

```diff
@@ -32,6 +32,31 @@ static int get_ext_windows(struct snd_sof_dev *sdev,
 	return 0;
 }
 
+static int get_cc_info(struct snd_sof_dev *sdev,
+		       struct sof_ipc_ext_data_hdr *ext_hdr)
+{
+	int ret;
+
+	struct sof_ipc_cc_version *cc =
+		container_of(ext_hdr, struct sof_ipc_cc_version, ext_hdr);
+
+	dev_info(sdev->dev, "Firmware info: used compiler %s %d:%d:%d%s used optimization flags %s\n",
```

Comment:
dev_dbg() ?

dev_info will be reported on most platforms, and it's noise for the average user

Reply:
Ok